web_src/css/repo.css

```
@@ -2349,14 +2349,10 @@ tbody.commit-list {
 .commit-body {
   margin: 0.25em 0;
   white-space: pre-wrap;
+  word-break: break-word;
```

```suggestion
  overflow-wrap: anyhwere;
```

Same effect and `word-break: break-word` is deprecated.

---

`overflow-wrap: anywhere;` vs `overflow-wrap: break-word;`, which is better?

---

Let's disuss in https://github.com/go-gitea/gitea/pull/30934.